packages/core/src/constants/type/QueryParams.ts

```diff
@@ -12,6 +12,7 @@ export interface QueryParams {
    */
   chainId?: ChainId
   isStatic?: boolean
+  refresh?: number | 'never' | 'everyBlock'
```

Comment:
```suggestion
  refresh?: Config['refresh']
```

We can tie the types into one place.